ubuntu-image

```diff
@@ -1,2 +1,28 @@
 #!/bin/sh
+
+if [ -n "$SNAP" ]; then
+    # If we're running as a snap, do some additional wrapping for classic
+    case $SNAP_ARCH in
+    amd64)
+        TRIPLET="x86_64-linux-gnu"
+        ;;
+    armhf)
+        TRIPLET="arm-linux-gnueabihf"
+        ;;
+    arm64)
+        TRIPLET="aarch64-linux-gnu"
+        ;;
+    *)
+        TRIPLET="$(uname -p)-linux-gnu"
+        ;;
+    esac
+
+    # We need to set the library paths for our snap to find the included libs.
+    export LD_LIBRARY_PATH=$SNAP/usr/lib:$SNAP/usr/lib/$TRIPLET:$SNAP/lib/$TRIPLET
+    export PYTHONPATH=$SNAP/lib/python3.5/site-packages:$PYTHONPATH
+    # Since we use some system-owned tools like mount, sudo or snap that we can't
+    # snappify, add some of the system paths to PATH
+    export PATH=$SNAP/usr/bin:$SNAP/bin:$SNAP/sbin:/usr/bin:/bin:$PATH
```

Comment:
Now I'm really annoyed. Super duper annoyed. When I tested all the changes last week on a vanilla clean instance I had this issue that PATH inside the snap was completely bollocks and had to be reconstructed from scratch. I thought this was just how things are in classic. And it was reproducible everytime.

Now, when I wanted to reproduce the same behavior it seems that now it behaves as it should, e.g. all the right paths are in PATH by default, as expected. It wasn't like that last week and I can even bet one of my limbs on it! I hate when such mysticism happens as then I lose liability. Eh.

Anyway, this can safely go away I guess.